server/admin/member.js

```diff
@@ -107,5 +128,9 @@ module.exports = {
     memberLeave: {
         endpoint: '/admin/member/leave',
         handler: createLeaveHandler
+    },
+    reap: {
+        endpoint: '/admin/reap',
+        handler: createReapHandler
```

Comment:
I don't understand the first comment. You can't call `/admin/reap` on faulty nodes. What this endpoint does, it speeds up the transition from faulty to tombstone (which by default takes 24h) for the node you're contacting. As soon as this happens, it will disseminate this to the entire cluster, causing everyone to reap their faulty nodes.